### DISCUSSION TITLE
Change keybind in config.kdl dont work

### DISCUSSION THREAD

**Discussion by oaSakamoto** [zellij-org/zellij#3615] [Q&A] [answered]
I find two references for "Ctrl h" in the config.kdl and change for "Ctrl m" and the keybind just work after i press "Ctrl h". Any idea?

**Comment by Lockszmith-GH** [comment, marked as answer]
Per the comment at the top of the [default configuration](https://github.com/zellij-org/zellij/blob/main/zellij-utils/assets/config/default.kdl) have you tried `clear-defaults=true` ?

**Reply by oaSakamoto** [reply]
Thanks, my mistake i don't read the comment in the top.

**Reply by Lockszmith-GH** [reply]
No worries, easy to miss that one. Glad that resolved the issue.
